src/analyzer/trackanalysisscheduler.cpp

```diff
@@ -275,7 +275,7 @@ bool TrackAnalysisScheduler::submitNextTrack(Worker* worker) {
         DEBUG_ASSERT(nextTrackId.isValid());
         if (nextTrackId.isValid()) {
             TrackPointer nextTrack =
-                    m_library->trackCollection().getTrackById(nextTrackId);
+                    m_library->trackCollections()->getTrackById(nextTrackId);
```

Comment:
I was confused by the naming. It sounds like a list of TrackCollection objects, but it's actually the track collection manager. I don't know if it's out-of-scope for this PR, but we should rename this to `trackCollectionManager()`.